contracts/solidity/oz/ERC-20-votes/erc20VotesTest.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+contract ERC20VotesTest is ERC20, ERC20Votes {
+    constructor(uint initialMintAmmount) ERC20("MyToken", "MYT") ERC20Permit("MyToken") {
+        _mint(msg.sender, initialMintAmmount);
+    }
+
+    function _afterTokenTransfer(address from, address to, uint256 amount)
+        internal
+        override(ERC20, ERC20Votes)
+    {
+        super._afterTokenTransfer(from, to, amount);
+    }
+
+    function _mint(address to, uint256 amount)
+        internal
+        override(ERC20, ERC20Votes)
+    {
+        super._mint(to, amount);
```

Comment:
internal _mint is no longer virtual. Not sure if it makes a difference here but please check with the latest OZ version